tracker/tracker_object.rst

```diff
@@ -653,3 +653,10 @@ Advanced Usage
             _paq.push(['customCrossDomainLinkVisitorIdGetter', function(url, name) {
                 return (new URL(url)).searchParams.get(name) || '';
             }]);
+
+.. function:: enableJSErrorTracking()
+
+    Enable tracking of unhandled JavaScript errors
```

Comment:
Since the description says what function "does" IMHO this will be more grammatically correct:
```suggestion
    Enables tracking of unhandled JavaScript errors
```